documentation/book/assembly-kafka-mirror-maker-bootstrap-servers.adoc

```diff
@@ -0,0 +1,21 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-bootstrap-servers-{context}']
+
+= Bootstrap servers
+
+Kafka Mirror Maker always works together with a two Kafka clusters (source and target).
+The source and the target Kafka clusters are specified in the form of two lists of bootstrap servers.
+On {ProductPlatformName}, each list must ideally contain the Kafka cluster bootstrap service which is named `_<cluster-name>_-kafka-bootstrap` and a port of 9092 for plain traffic or 9093 for encrypted traffic.
+
+The list of bootstrap servers can be configured in the `bootstrapServers` property in `KafkaMirrorMaker.spec.consumer` and `KafkaMirrorMaker.spec.producer`. The servers should be a comma-separated list containing one or more Kafka brokers or a service pointing to Kafka brokers specified as a `<hostname>:<port>` pairs.
+
+When using Kafka Mirror Maker with a Kafka cluster not managed by {ProductName}, you can specify the bootstrap servers list according to the configuration of a given cluster.
```

Comment:
"according to the configuration of the given cluster." (the instead of a, otherwise I'd not sure how to parse this sentence)